Sources/Subs.php

```diff
@@ -4982,9 +4983,31 @@ function inet_dtop($bin)
 	if ($db_type == 'postgresql')
```

Comment:
There is no need to test, when the output came from pg
because your are not able to save wrong data in pg-> no need for checks.